### PR TITLE
Adds tree intersection challenge 32, updates table of contents.

### PR DESCRIPTION
**Pull request by ryanemmans**
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
       - [x] Summary, Description, Approach & Efficiency, Solution
       - [x] Picture of whiteboard
       - [x] Link to code
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [x] Expected failure
       - [ ] Edge Case (if applicable/obvious)

### CODE REVIEW

File: javascript/401/class-32-tree-intersection/tree-intersection.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+function tree_intersection(bt1, bt2) {
+  // Coming Soon...
+}
+
+module.exports = repeatedWord;
```
Comment (adrianhuebner) [inline javascript/401/class-32-tree-intersection/tree-intersection.js:7]: I would like to see code and tests added to this!
